src/node/in_memory.rs

```diff
@@ -795,7 +786,38 @@ impl<S: std::fmt::Debug + ForkSource> InMemoryNodeInner<S> {
         let tx: Transaction = l2_tx.into();
         vm.push_transaction(tx);
 
-        vm.execute(InspectExecutionMode::OneTx)
+        let call_tracer_result = Arc::new(OnceCell::default());
+        let error_flags_result = Arc::new(OnceCell::new());
+        let bootloader_debug_result = Arc::new(OnceCell::default());
+
+        let tracers = vec![
+            CallErrorTracer::new(error_flags_result.clone()).into_tracer_pointer(),
+            CallTracer::new(call_tracer_result.clone()).into_tracer_pointer(),
+            BootloaderDebugTracer {
+                result: bootloader_debug_result.clone(),
+            }
+            .into_tracer_pointer(),
+        ];
+        // TODO: Ask if vm.inspect is the same as vm.execute
+        let tx_result = vm.inspect(&mut tracers.into(), InspectExecutionMode::OneTx);
```

Comment:
@itegulov do you know if calling `vm.inspect` vs `vm.execute` makes any difference? It doesnt seem so but wanted to confirm.